### DISCUSSION TITLE
C# just got unsigned right shift operator

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#804] [General]
https://github.com/dotnet/csharplang/blob/main/proposals/unsigned-right-shift-operator.md

The `>>>`  triple-symbol monster seems able to scare even the most battle-hardened VB developers! :-))

Should we consider `URShift` as a future feature request to mitigate this? :-))

**Comment by mansellan** [comment]
Uhm, `>>>=` might be a step too far even for me :-)
